pygmt/figure.py

```diff
@@ -289,6 +289,9 @@ def savefig(
         dpi : int
             Set raster resolution in dpi. Default is 720 for PDF, 300 for
             others.
+        **kwargs : dict
+            Additional keyword arguments passed to
+            :meth:`pygmt.Figure.psconvert`.
```

Comment:
Unsure if we should use `kwargs` or `**kwargs` here.

Reply:
I had a look at some projects and there seems to be a mix 😅. Let's go with `**kwargs` following https://docs.xarray.dev/en/v2022.06.0/generated/xarray.open_dataset.html, unless anyone else has a good reason on why not.